dotcom-rendering/src/web/components/Counts.stories.tsx

```diff
@@ -79,36 +96,152 @@ export const ShareOnly = () => {
+			<Counts format={{ ...format, theme: ArticlePillar.Sport }}>
+				<div className="meta-number">
+					<ShareCount
+						ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{ ...format, theme: ArticlePillar.Sport }}
+					/>
+				</div>
+				<div className="meta-number">
+					<CommentCount
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{ ...format, theme: ArticlePillar.Sport }}
+					/>
+				</div>
+			</Counts>
+			<Counts format={{ ...format, theme: ArticlePillar.Lifestyle }}>
+				<div className="meta-number">
+					<ShareCount
+						ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{ ...format, theme: ArticlePillar.Lifestyle }}
 					/>
 				</div>
 				<div className="meta-number">
 					<CommentCount
-						isCommentable={false}
-						commentCount={239}
-						palette={decidePalette(format)}
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{ ...format, theme: ArticlePillar.Lifestyle }}
+					/>
+				</div>
+			</Counts>
+			<Counts format={{ ...format, theme: ArticlePillar.Opinion }}>
+				<div className="meta-number">
+					<ShareCount
+						ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{ ...format, theme: ArticlePillar.Opinion }}
+					/>
+				</div>
+				<div className="meta-number">
+					<CommentCount
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{ ...format, theme: ArticlePillar.Opinion }}
+					/>
+				</div>
+			</Counts>
+			<Counts format={{ ...format, theme: ArticleSpecial.Labs }}>
+				<div className="meta-number">
+					<ShareCount
+						ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{ ...format, theme: ArticleSpecial.Labs }}
+					/>
+				</div>
+				<div className="meta-number">
+					<CommentCount
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{ ...format, theme: ArticleSpecial.Labs }}
+					/>
+				</div>
+			</Counts>
+			<Counts format={{ ...format, theme: ArticleSpecial.SpecialReport }}>
+				<div className="meta-number">
+					<ShareCount
+						ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{
+							...format,
+							theme: ArticleSpecial.SpecialReport,
+						}}
+					/>
+				</div>
+				<div className="meta-number">
+					<CommentCount
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{
+							...format,
+							theme: ArticleSpecial.SpecialReport,
+						}}
```

Comment:
I've deleted `CommentCount.stories` and replaced that coverage here in `Counts.stories`